### PR TITLE
Fix some typos and Spanish translations

### PR DESCRIPTION
**Pull request by luisantoniocrag**

### CODE REVIEW

**Comment by m52go** [comment]
@MwithM would you be able and/or willing to review this?

File: _data/es.yml [hunk 1]
```diff
@@ -274,26 +274,26 @@ faq:
...
   hBisqDaoHow: "¿Cómo funciona la DAO Bisq?"
   pBisqDaoHow: >
-    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrageia.</p> <ul>
+    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrategia.</p> <ul>
         <li>Las comisiones de intercambio se distribuyen directamente desde los comerciantes a los contribuyentes con un token (bitcoin coloreado) llamado BSQ.</li>
         <li>La estrategia se determina colectivamente a través de la votación hecha en el software Bisq y registrada en la red Bitcoin.</li>
     </ul> <p>Aprenda más acerca de cómo funciona en realidad con <a href='https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e' target='_blank'>esta serie de videos cortos de 3-5 minutos</a>.</p>
-  hNeedToKnow: "¿Como comerciante, qué necesito saber acerca de la DAO?"
+  hNeedToKnow: "¿Cómo comerciante, qué necesito saber acerca de la DAO?"
```
Comment (MwithM) [inline _data/es.yml]: The original, without accent, was fine.

"Como" en este caso, aunque sea pregunta, no tiene valor interrogativo, con lo que va sin acento (preposición). https://www.fundeu.es/recomendacion/como-con-sin-tilde-diferencia/

**Comment by MwithM** [comment]
Great job Luis Antonio, I've seen the 4 files changed and all the corrections are good, except a very small one.

File: _data/es.yml [hunk 2]
```diff
@@ -274,26 +274,26 @@ faq:
...
   hBisqDaoHow: "¿Cómo funciona la DAO Bisq?"
   pBisqDaoHow: >
-    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrageia.</p> <ul>
+    <p>Los propósitos primarios de la DAO Bisq son financiar el proyecto y determinar la estrategia.</p> <ul>
         <li>Las comisiones de intercambio se distribuyen directamente desde los comerciantes a los contribuyentes con un token (bitcoin coloreado) llamado BSQ.</li>
         <li>La estrategia se determina colectivamente a través de la votación hecha en el software Bisq y registrada en la red Bitcoin.</li>
     </ul> <p>Aprenda más acerca de cómo funciona en realidad con <a href='https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e' target='_blank'>esta serie de videos cortos de 3-5 minutos</a>.</p>
-  hNeedToKnow: "¿Como comerciante, qué necesito saber acerca de la DAO?"
```
Comment (MwithM) [inline _data/es.yml:291]:
```Suggestion:
hNeedToKnow: "¿Como comerciante, qué necesito saber acerca de la DAO?"
``` 

"Como" no es pregunta en este caso, es preposición.
https://www.fundeu.es/recomendacion/como-con-sin-tilde-diferencia/

Comment (luisantoniocrag) [reply _data/es.yml:291]: Gracias por tomarte el tiempo para revisar @MwithM

File: _data/es.yml (hunk 1 above)
Comment (m52go) [inline _data/es.yml]:
```suggestion
  hNeedToKnow: "¿Como comerciante, qué necesito saber acerca de la DAO?"
```

Comment (m52go) [reply _data/es.yml]: @luisantoniocrag looks like this was the only change MwithM suggested...would you mind committing it?

**Pull request merged**